site/config/target_env.md

```diff
@@ -0,0 +1,93 @@
+## Target Environments
+
+The [Discover Domain](../discover.md) and [Prepare Model](../prepare.md) Tools allow you to customize the model and other files produced to be compatible with a specific target environment. Options for a target environment may include:
+- Using model tokens for some attributes in the model (see [Model Tokens](../../README.md#model-tokens))
+- Using Kubernetes secrets for credentials in the model
+- Apply filters to the model (see [Model Filters](../tool_filters.md))
```

Comment:
Apply  -> Applying (to be consistent (parallel construction) with the other bulleted items)